JotunnLib/Utils/SimpleJVL.cs

```diff
@@ -0,0 +1,217 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using UnityEngine;
+using Jotunn.Managers;
+using Jotunn.Entities;
+using Jotunn.Configs;
+
+namespace Jotunn.Utils
+{
+    public static class SimpleJVL
+    {   // Simple JVL - For simpletons, a language simplifier
+        public static AssetBundle assetBundle;
+        public static GameObject prefabObject;
+        public static ItemDrop.ItemData itemDrop;
+        public static CraftingStation reflect;
+        public static string needs;
+        public static int needsAmount;
+        public static int needsAmountPerLevel;
+        public static bool needsRecovery;
+        public static int amount;
+        public static int amountPerLevel;
+        public static void LoadAssets(string assetBundleName)
+        {
+            AssetBundle.UnloadAllAssetBundles(false);
```

Comment:
This will unload all asset bundles, potentially from other mods too, why is it there ?

Reply:
To prevent the error from trying to load multiple assetbundles while previous bundles are still loaded, how would I fix this to simply unload the current bundle if a bundle is already loaded?